addition_1.txt

```diff
@@ -1 +1 @@
-This is a file added to a new branch one for test purposes
```

Comment:
Adding a comment to the initial commit. Let's see how that works.
So, I've added a test file.

Reply:
These were the merge/branch/whatever comments, you moron. Not the commit comments.